src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  &__logo {
+    display: flex;
+    align-items: center;
+
+    &-image {
+      width: 40px;
+      height: 40px;
+    }
+  }
+
+  &__nav {
+    &-list {
+      display: flex;
+      list-style: none;
+      padding: 0;
+      margin: 0;
+    }
+    &-link {
+      margin-left: 20px;
+
+      &-text {
+        line-height: 60px;
+        font-weight: 500;
+        font-size: 12px;
+        text-decoration: none;
+        color: #060b35;
+        text-transform: uppercase;
+        display: block;
+      }
+    }
+    
+    &-link:first-child {
+      margin-left: 0;
+    }
+
+    [data-qa='nav-hover']:hover {
+      transition: all 0.3s;
+
+      color: #00acdc;
+    }
```

Comment:
The selector [data-qa='nav-hover']:hover is not specific enough. Consider using a more specific class or element selector to avoid potential conflicts with other elements that might use the same data attribute.